api/protobuf-spec/messages.proto

```diff
@@ -20,6 +20,31 @@ message MatchObject{
   repeated Roster rosters = 4;          // Rosters of players.  
   repeated PlayerPool pools = 5;        // 'Hard' filters, and the players who match them.  
   string status = 6;                    // Resulting status of the match function
+  string name = 7;                      // Arbitrary developer-chosen, human-readable string.  Optional.
+
+}
+
+message MmfSpec {
+    string name = 1;  // Arbitrary developer-chosen, human-readable string.  Optional.
+    string host = 2;  // Host or DNS name for service providing this MMF.  Must be resolve-able by the backend API.
+    int32 port = 3;   // Port number for service providing this MMF.
```

Comment:
I think we'll want to revisit this. There might be a security concern here. It's probably better to configure the routing via ConfigMap. For now just add a TODO.

Reply:
The expectation is that hosts and ports for served MMFs are running using ClusterIP and can't be accessed by anything running outside the cluster, so I think it's probably fine?